Client/Frontend/Browser/URLBarView.swift

```diff
@@ -424,12 +424,10 @@ class URLBarView: UIView {
             // Clear any existing text, focus the field, then set the actual pasted text.
             // This avoids highlighting all of the text.
             locationTextField.text = ""
-            locationTextField.becomeFirstResponder()
```

Comment:
Does `SearchTest` -> `testURLBarContextMenu` still pass with this change? I think the order here was important...if we become first responder after setting the text, it might highlight the whole string instead of the pasted portion.